### ISSUE TITLE
PassPlanFilePlugin & TagPlugin - Error: Can't set variables when applying a saved plan

### ISSUE THREAD

**Issue by vincentclee** [manheim/terraform-pipeline#412]
The PassPlanFilePlugin & TagPlugin are not compatible with each other. When the TagPlugin is enabled, terraform pipeline will need to remove the `-var` option on apply.

```
 + terraform apply -input=false -auto-approve '-var=tags={"Environment":"ci1",...}' tfplan-ci1
 ╷
 │ Error: Can't set variables when applying a saved plan
 │ 
 │ The -var and -var-file options cannot be used when applying a saved plan
 │ file, because a saved plan includes the variable values that were set when
 │ it was created.
 ╵
```

**Comment by kmanning** [comment]
TagPlugin has a `disableOnApply` method, to solve this conflict.  https://github.com/manheim/terraform-pipeline/blob/master/src/TagPlugin.groovy#L77-L81

See Issue #296, which was resolved and released in v5.11 - https://github.com/manheim/terraform-pipeline/releases/tag/v5.11

**Comment by vincentclee** [comment]
Thanks @kmanning Great feature.
